source/operator.rst

```diff
@@ -62,6 +70,12 @@ The spec part of the `deploy/cr.yaml <https://github.com/percona/percona-server-
      - The (non-standard) cluster domain to be used as a suffix of the Service
        name
 
+   * - clusterServiceDNSMode
+     - string
+     - ``Internal``
+     - Can be either ``Internal`` (use ``pod.statefulset.namespace.clusterServiceDNSSuffix`` internal DNS name for non-exposed MongoDB instances, and use internal IP for instances exposed with ClusterIP), or ``ServiceMesh`` (use ``service.namespace.clusterServiceDNSSuffix`` Service :abbr:`FQDN (fully qualified domain name)` when generating host names for exposed MongoDB instances).
```

Comment:
We need to mention this option only takes effect if replset is exposed.

Also what it does seems unclear to me. To summarize from my end:

When you expose the replset the operator configures nodes to use ClusterIP addresses. If you want to use FQDN of the exposed services you need to change `clusterServiceDNSMode` to `ServiceMesh`.